doc/classes/File.xml

```diff
@@ -317,7 +337,8 @@
 			</argument>
 			<description>
 				Stores an integer as 64 bits in the file.
-				[b]Note:[/b] The [code]value[/code] must lie in the interval [code][-2^63, 2^63 - 1][/code] (i.e. be a valid [int] value).
+				[b]Note:[/b] The [code]value[/code] must lie in the interval [code][-2^63, 2^63 - 1][/code] (i.e. be a valid [int] value). Any other value will overflow and wrap around.
+				To store a signed integer, use [method store_64], or convert it manually (see [method store_16] for an example).
```

Comment:
> Any other value will overflow and wrap around.

There can be no 'other value' in GDScript. (What's in C++ I'm not sure.)
![image](https://user-images.githubusercontent.com/47700418/84180191-c461a400-aa8f-11ea-957d-914cf65735c0.png)

> To store a signed integer, use [method store_64]

Cool recursion. :smile:

Reply:
Whoops, will fix.